### ISSUE TITLE
Blog post error when base-url in /

### ISSUE THREAD

**Issue by akhilerm** [jekyller/sustain#19]
When the base-url is given as `/` the links of the generated posts will be wrong. `//post1.html` instead of `/post1.html`. Can be fixed by removeing `{{ site.baseurl }}` from `blog.html`

**Comment by Akronix** [comment]
I think this is due to sustain was programmed for an older version of jekyll? Well, in any case, for me it worked to set `baseurl: ""`.

**Comment by akhilerm** [comment]
@Akronix  If `baseurl: ""` is set as empty, will the home button link work?

**Comment by Akronix** [comment]
Definitely it should. It does for me: https://akronix.github.io/
 but I have set the home button to show the blog page: 
https://github.com/Akronix/akronix.github.io/commit/d35226102bad1a190414972bc20d8c8341618b67#diff-822e9cf54ee310f9a7ebd0d16c9ae066
Doesn't it work for you?

**Comment by akhilerm** [comment]
Nope, it its not working for me when baseurl is set to empty. I think this ha something to do with different versions of jekyll

**Comment by Akronix** [comment]
Hmm Indeed I think I'm using jekyll 3.0 and I think sustain was made for an older version of jekyll.
However, what I did is to set href="/" when I want the link to point to the home page, as you can see in my "About" element in the header.

**Comment by akhilerm** [comment]
Yeah. I checked. Shall we change the code and update it to work with latest version? @Akronix

**Comment by Akronix** [comment]
That sounds cool, however I suffer from a very severe lack of time. what does the maintainer think about this @biomadeira?

**Comment by biomadeira** [comment]
@Akronix and @akhilerm I will have look later tonight ;)
